docs/rst/manual/reference/workflows/built_in.rst

```diff
@@ -213,43 +213,6 @@ optional arguments a range and/or list of which realizations to run.
 
 	ENSEMBLE_RUN 1-5, 8
 
-**LOAD_RESULTS**
```

Comment:
Not sure I understand it deeply enough, but why this workflow is to be removed?

Reply:
Because it is (probably) never used.